LICENSE.rst

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
can you save this as `LICENSE` (without the `.rst`)?

Reply:
sure

Reply:
done @wolfv